CorvallisBus.Core/Models/Connexionz/ConnexionzRoute.cs

```diff
@@ -23,10 +23,33 @@ public ConnexionzRoute(RoutePatternProjectRoute routePatternProjectRoute)
 
             Polyline = EncodePolyline(GetPoints(longestPattern.Mif));
 
+            var points = GetPoints(longestPattern.Mif);
+            // TODO: get the actual data fixed
+            if (RouteNo == "4")
+            {
+                var pointsList = points.ToList();
+
+                // there's a weird point on philomath blvd messing up the route path. just filter it out.
```

Comment:
Can we linq this and/or write a private method that is named to convey what's happening to get rid of that comment?

Reply:
I mentioned this to you offline but basically I'm going to leave it as-is in the hope that the data will get fixed and I can just delete it.